documentation/src/docs/auth/client-auth.md

```diff
@@ -0,0 +1,35 @@
+<!--
+Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+-->
+# Client Auth interface
```

Comment:
Please mention a consistent format for: _auth_ Interface or _Auth_ Interface
Client _Auth_ Interface